realm/realm-library/src/main/java/io/realm/RealmResults.java

```diff
@@ -851,12 +851,12 @@ void setPendingQuery(Future<Long> pendingQuery) {
     }
 
     /**
-     * Returns {@code true} if the results are not yet loaded, {@code false} if they are still loading. Synchronous
+     * Returns {@code false} if the results are not yet loaded or still being loaded. Synchronous
```

Comment:
Returns {@code true} if the results are loaded and available, {@code false} if they are still loading.